### PR TITLE
Clean the coupling interface (remove deletePlus, clean code)

### PR DESCRIPTION
**Pull request by dklein-pik**
## :bird: Description of this PR :bird:

- Clean the coupling interface 

## :wrench: Checklist for PR creator :wrench:

- [x] Label pull request [from the label list](https://github.com/magpiemodel/magpie/labels).

- [x] Self-review own code
 
- [x] Document changes 

- [x] Perform test runs

### :chart_with_downwards_trend: Performance changes :chart_with_upwards_trend:
  
  - Current develop branch default : ** mins
  - This PR's default :  ** mins

## :rotating_light: Checklist for reviewer :rotating_light:

- PR is labeled correctly
- Code changes look reasonable
  - No hard coded numbers and cluster/country/region names.
  - No unnecessary increase in module interfaces
  - model behavior/performance is satisfactory.
- Changes are properly documented
  - `CHANGELOG` is updated correctly
  - Updates in interfaces have been properly addressed in the module documentations
  - In-code documentation looks appropriate

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,21 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+## Unreleased
+
+### changed
+-
+
+### added
+-
+
+### removed
+- start_functions: coupling interface: remove remind2::deletePlus
```
Comment (k4rst3ns) [inline CHANGELOG.md:18]: changed to:
`-  **scripts** removed remind2::deletePlus in coupling interface of start_function`

**Review by k4rst3ns — CHANGES_REQUESTED** [review]
Contentwise this looks good to me.
I guess RSE reviewer might be more advanced in understanding the change to the start_function ^^'

**Review by k4rst3ns — APPROVED** [review]
Changelog good to go now.

**Comment by pfuehrlich-pik** [comment]
Woah wait, this is a PR onto the master branch which is reserved to releases, right?

**Comment by pfuehrlich-pik** [comment]
We will need to revert this. Can you please re-create this as a PR onto develop @dklein-pik ?

**Pull request merged**